bucket/xdman.json

```diff
@@ -1,21 +1,42 @@
 {
-    "version": "7.2.8",
+    "version": "7.2.11",
     "description": "A powerful FOSS downloader supports protocols as HTTP,HTTPS,FTP,MPEG-DASH,HLS,HDS and features as seamlessly integration with browsers, monitoring and capture, broken/dead tasks resumption",
-    "homepage": "http://xdman.sourceforge.net/",
-    "license": "Unknown",
-    "url": "https://downloads.sourceforge.net/project/xdman/xdmsetup-2018.msi",
-    "hash": "sha1:506c8b1014aa1ea17979c548aa7c4dd1bd110f14",
-    "extract_dir": "XDM",
-    "pre_install": "Set-Content \"$dir\\xdman.bat\" '@start javaw.exe -jar \"%~dp0xdman.jar\" %*' -Encoding Ascii",
-    "bin": "xdman.jar",
+    "homepage": "https://subhra74.github.io/xdm",
+    "suggest": {
+        "java": [
+            "java/openjdk11"
+        ],
+        "ffmpeg": [
+            "main/ffmpeg",
+            "main/ffmpeg-shared"
+        ],
+        "youtube-dl": [
+            "main/youtube-dl"
+        ]
+    },
+    "license": "GPL-2.0-only",
+    "url": "https://github.com/subhra74/xdm/releases/download/7.2.11/xdman.jar",
+    "hash": "8117f286f9de1e57f4551676d8f0ab3e2e990444744b595f7cc4cb9e01f51c7c",
+    "pre_install": [
+        "Set-Content \"$dir\\xdman.bat\" '@start javaw.exe -jar \"%~dp0xdman.jar\" %*' -Encoding Ascii"
+    ],
+    "post_install": [
+        "cmd /c '@echo off && mklink \"%AppData%\\..\\..\\scoop\\apps\\xdman\\current\\ffmpeg.exe\" \"%AppData%\\..\\..\\scoop\\shims\\ffmpeg.exe\" >nul 2>&1'",
+        "cmd /c '@echo off && mklink \"%AppData%\\..\\..\\scoop\\apps\\xdman\\current\\ffmpeg.shim\" \"%AppData%\\..\\..\\scoop\\shims\\ffmpeg.shim\" >nul 2>&1'",
+        "cmd /c '@echo off && mklink \"%AppData%\\..\\..\\scoop\\apps\\xdman\\current\\youtube-dl.exe\" \"%AppData%\\..\\..\\scoop\\shims\\youtube-dl.exe\" >nul 2>&1'",
+        "cmd /c '@echo off && mklink \"%AppData%\\..\\..\\scoop\\apps\\xdman\\current\\youtube-dl.shim\" \"%AppData%\\..\\..\\scoop\\shims\\youtube-dl.shim\" >nul 2>&1'"
```

Comment:
- Completely nonsense assumption that everyone has scoop userprofile and ignoring global installations
- Absolutely ugly solution to "problem" which noone came in

Reply:
Unfortunately, xdman doesn't use exe files from global PATH :/
I couldn't find any way to solve this except using an absolute path to exe files
What is your suggestion to solve this "problem" ?

Reply:
> * Completely nonsense assumption that everyone has scoop userprofile and ignoring global installations
> * Absolutely ugly solution to "problem" which noone came in

Unfortunately, xdman doesn't use exe files from global PATH :/
I couldn't find any way to solve this except using an absolute path to exe files
What is your suggestion to solve this "problem" ?

Reply:
btw, the only difference between the two setup files is the FFmpeg version (xdmsetup.msi uses master branch version)